build.gradle

```diff
@@ -20,5 +20,17 @@ subprojects { project ->
     options.compilerArgs += ['-Xlint:all', '-Xlint:-path', '-Werror']
   }
 
+  configurations.all {
+    resolutionStrategy {
+      eachDependency { details ->
+        // Force all the error-prone dependencies to use the same version.
+        if (details.requested.group == 'com.google.errorprone' &&
+                details.requested.name.startsWith('error_prone_')) {
+          details.useVersion '2.1.2'
```

Comment:
could specify in the dependencies, if you think this is overkill for this project.